.eslintignore

```diff
@@ -1 +0,0 @@
-bower_components
```

Comment:
This isn't used since an explicit list of files to lint is passed to the `lint` npm script.